.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint and Test
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x,22.x]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install dependencies
+      run: npm ci
```

Comment:
Hi @BRIHMAT-Naoui , I guess we don't have used CI integration yet and this command will not work. To install dependencies we run this command:

`npm run install`